aspnetcore/security/anti-request-forgery.md

```diff
@@ -49,7 +49,7 @@ This example requires the user to click the form button. The malicious page coul
 
 Using SSL doesn't prevent a CSRF attack, the malicious site can send an `https://` request. 
 
-Some attacks  target site endpoints that respond to `GET` requests, in which case an image tag can be used to perform the action (this form of attack is common on forum sites that permit images but block JavaScript). Applications that change state with `GET` requests are  vulnerable from malicious attacks.
+Some attacks target site endpoints that respond to `GET` requests, in which case an image tag can be used to perform the action (this form of attack is common on forum sites that permit images but block JavaScript). Applications that change state with `GET` requests are vulnerable from malicious attacks.
```

Comment:
Highlight that this is a violation of the HTTP spec. This also doesn't scream DON'T - make it scream DON'T DO THIS